german/the-black-hack.md

```diff
@@ -101,7 +101,7 @@ The Black Hack verwendet keine Rettungswürfe und lässt die Spieler stattdessen
 
 |              INT               |            WEI            |           CHA           |
 |:------------------------------:|:-------------------------:|:-----------------------:|
-| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Geistige Beeinflussung. |
+| Zaubern und Magie widerstehen. | Täuschung und Illusionen. | Bezauberung. |
```

Comment:
Bezauberung ist eigentlich auch falsch, aber ich hatte in meiner Runde das Problem unsicher zu sein ob der Zauber Charm/Bezauberung nun unter INT oder CHA fällt. Kreative Ideen?